apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -21,15 +21,15 @@ namespace Google.Cloud.Spanner.Data.IntegrationTests
 {
     public class TestDatabaseFixture : IDisposable {
 
-        private readonly string _databaseName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
+        private readonly string _databaseName = "scratch";// "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
```

Comment:
Right, I made the tests create a brand new database, but found it too painfully slow to iterate on.
I'll remove this for now.